tests/test_fastai.py

```diff
@@ -27,8 +27,9 @@ def test_tabular(self):
             "/input/tests/data/train.csv",
             cont_names=["pixel"+str(i) for i in range(784)],
             y_names='label',
-            procs=[FillMissing, Categorify, Normalize])       
+            procs=[FillMissing, Categorify, Normalize])
         learn = tabular_learner(dls, layers=[200, 100])
-        learn.fit_one_cycle(n_epoch=1)
+        with learn.no_bar():
```

Comment:
fastprogress was trying to render html while running in the background and it caused issues, so I turned off the progress bar rendering.